lib/gateways/sessions.rb

```diff
@@ -28,6 +28,10 @@ def search(username: nil, ips: nil)
       end
     end
 
+    def count_distinct_users
```

Comment:
I would name this `unique_user_count`, it fits more with what the user would see. "distinct" is really just a ruby method. Its also common practice to have the `_count` but last in ruby.